bundles/org.openhab.core.io.rest.auth/src/main/java/org/openhab/core/io/rest/auth/internal/AuthFilter.java

```diff
@@ -36,9 +40,11 @@
  * @author Yannick Schaus - initial contribution
  */
 @PreMatching
+@Component(configurationPid = "org.openhab.rest.auth")
```

Comment:
All our configuration pids should be of format `org.openhab.<scope>`, so here I would suggest `org.openhab.auth`.

Reply:
With a couple of exceptions ;)

https://github.com/openhab/openhab-core/blob/0182ce567d7a74046a0025a0f3d6629232321209/bundles/org.openhab.core.storage.json/src/main/java/org/openhab/core/storage/json/internal/JsonStorageService.java#L41

Looks like `org.openhab.auth` is already used:

https://github.com/openhab/openhab-core/blob/61e17ce39e83daa7f1d8c41512dead8d95d6e5d9/bundles/org.openhab.core.io.http.auth/src/main/java/org/openhab/core/io/http/auth/internal/AuthenticationHandler.java#L47

Reply:
> With a couple of exceptions ;)

You only list one and as we say in German: "an exception proves the rule" ;-). We can fix that easily, I'd say.

> Looks like org.openhab.auth is already used

Then we should see how we can merge the functionality - for a user, there should in the end only be one `conf/services/auth.cfg` file to do configuration around auth.
Btw, where is this class here at all taking a configuration? Am I missing something?

Reply:
> Then we should see how we can merge the functionality 

We can use the same configurationPid for several services? (honest question, I'm not an expert).

> Btw, where is this class here at all taking a configuration? Am I missing something?

Nowhere for now, I added it in advance for when we'd like to add some like "protect the entire API, not only the admin operations".

Reply:
All things considered, I removed it for now since it's not used.

Reply:
> We can use the same configurationPid for several services? (honest question, I'm not an expert).

I cannot say - I remember different things from the past and would have to dig into a suitable solution myself.

> I removed it for now since it's not used.

Sounds like the most elegant solution for the moment! 😎